packages/block-library/src/group/icon.js

```diff
@@ -4,5 +4,5 @@
 import { Path, SVG } from '@wordpress/components';
 
 export default (
-	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><Path d="M19 12h-2v3h-3v2h5v-5zM7 9h3V7H5v5h2V9zm14-6H3c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h18c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 16.01H3V4.99h18v14.02z" /><Path d="M0 0h24v24H0z" fill="none" /></SVG>
+	<SVG width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><Path fill-rule="evenodd" clip-rule="evenodd" d="M9 8a1 1 0 0 1 1-1h6a1 1 0 0 1 1 1v4a1 1 0 0 1-1 1h-1v3a1 1 0 0 1-1 1H8a1 1 0 0 1-1-1v-4a1 1 0 0 1 1-1h1V8zm2 3h4V9h-4v2zm2 2H9v2h4v-2z" /><Path fill-rule="evenodd" clip-rule="evenodd" d="M2 4.732A2 2 0 1 1 4.732 2h14.536A2 2 0 1 1 22 4.732v14.536A2 2 0 1 1 19.268 22H4.732A2 2 0 1 1 2 19.268V4.732zM4.732 4h14.536c.175.304.428.557.732.732v14.536a2.01 2.01 0 0 0-.732.732H4.732A2.01 2.01 0 0 0 4 19.268V4.732A2.01 2.01 0 0 0 4.732 4z" /></SVG>
```

Comment:
`fill-rule` and `clip-rule` are not valid names for props. When running with the [`SCRIPT_DEBUG` constant](https://wordpress.org/support/article/debugging-in-wordpress/#script_debug), you will see errors log to the developer tools console:

>Warning: Invalid DOM property `fill-rule`. Did you mean `fillRule`?

>Warning: Invalid DOM property `clip-rule`. Did you mean `clipRule`?

(The message includes the respective solution for each)

React uses DOM _property_ names for elements, not the HTML element _attribute_ names. This is the same reason we assign classes as `className` and not `class`.

See also:

- https://reactjs.org/docs/dom-elements.html
- https://reactjs.org/blog/2017/09/08/dom-attributes-in-react-16.html

Reply:
Thanks for catching that! I've submitted #16096 to correct the issue.